modules/planning/tasks/deciders/path_assessment_decider.cc

```diff
@@ -41,6 +41,8 @@ using apollo::hdmap::HDMapUtil;
 // PointDecision contains (s, PathPointType, distance to closest obstacle).
 using PathPointDecision = std::tuple<double, PathData::PathPointType, double>;
 
+constexpr double kMinObstacleArea = 1e-4;
```

Comment:
This is a global variable. Mark either `static` or anonymous namespace to make it local to this file.

Reply:
This is a global variable but in .cc only.
Others won't include a .cc file.

Reply:
I added anonymous namespace anyway.